### ISSUE TITLE
Feed Activity needs a ListView for posts

### ISSUE THREAD

**Issue by brian-murphy** [git-mad/Bitter#1]
Right now, the FeedActivity is looking a little bland. We need it to have more functionality than just saluting the world. I'm thinking that a ListView is in order. Thoughts?

**Comment by brian-murphy** [comment]
[THOUGHT]

**Comment by myozka** [comment]
RecycleView?

**Comment by KPrabs106** [comment]
I like CardView

**Comment by brian-murphy** [comment]
Yeah, CardView will probably be better suited for this kind of thing. I'll plug it in.

**Comment by myozka** [comment]
I think this is done magnificently with a CardView inside RecyclerView

**Comment by myozka** [comment]
I think this is done some time ago. (With CardView inside RecyclerView).
